var/spack/repos/builtin/packages/wrf-io/package.py

```diff
@@ -9,14 +9,21 @@
 class WrfIo(CMakePackage):
     """The WRFIO package is a lightweight WRF-IO API library for Unified
     Post Processor (UPP). It reads wrf forecasts (WRF state plus
-    diagnostics)."""
+    diagnostics).
+
+    This is part of the NCEPLIBS project."""
 
     homepage = "https://noaa-emc.github.io/NCEPLIBS-wrf_io"
     url      = "https://github.com/NOAA-EMC/NCEPLIBS-wrf_io/archive/refs/tags/v1.2.0.tar.gz"
 
-    maintainers = ['t-brown']
+    maintainers = ['t-brown', 'kgerheiser', 'Hang-Lei-NOAA', 'edwardhartnett']
 
     version('1.2.0', sha256='000cf5294a2c68460085258186e1f36c86d3d0d9c433aa969a0f92736b745617')
 
-    depends_on('netcdf-c')
+    variant('openmp', default=False, description='Enable multithreading with OpenMP')
```

Comment:
Why remove `NetCDF-c` as a dependency?  It is still being listed as a prerequisite in the documentation.

Reply:
By documentation I mean the `README.md` at https://github.com/NOAA-EMC/NCEPLIBS-wrf_io.

Reply:
Because it is already a requirement of netcdf-fortran.

Reply:
Okay.  I was expressing a concern about the inconsistency.

Reply:
Which was the exact reason I had it listed. @edwardhartnett  would you like me to open an upstream PR with `s/netcdf C/netcdf Fortran/` for the documentation?

Reply:
@t-brown no I don't think that's a good idea. Some package require netcdf-c, some netcdf-c and netcdf-fortran.

Reply:
I'm not following. You've removed the `netcdf C` dependency in this package, in looking at the `CMakeLists.txt` there is only a requirement for Fortran.

```
find_package(NetCDF REQUIRED COMPONENTS Fortran)
```

I'm agreeing with  @tldahlgren that this is inconsistent and I would like to clear that up.

Reply:
Whenever netcdf-fortran is used, netcdf-c is required. So the netcdf-c dependency is still there. I will return it to the package file...